### ISSUE TITLE
MERLIN: jaws issue

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#4953]
Email from Adroja on Merlin:

> In slide 4 jaws were v49x49mm and last cycle and this cycle runs show the same flux on the sample.
In side 5 Horizontal Jaws=49mm fixed) (1, left)  jaws top=2mm and bottom -24.6mm, (2, right)  top=24.6mm and bottom =-2mm.
These two runs should be same, but not. The counts for option (2) are less. This indicates that center of jaws has moved up.

With follow-up from Rob B:

> Has the software been changed or updated?. There was always an offset on the jaws which was programmed in the software, I seem to remember I had similar problems once and it was because the jaw software had been changed,

They will probably want us to investigate to see what, if anything, is wrong. At the moment from the emails above it's not entirely clear to me what the problem is (especially as the referenced slides were not attached).

**Comment by Tom-Willemsen** [comment]
Met with Jamie Nutter on MERLIN.

Together we measured the motion of the jaws, and found that setting gaps smaller than 5mm caused the jaws to physically clash.

After various measurements were taken, we changed the following offsets:

| Axis | Old offset | New offset |
| -- | -- | -- |
| North (J1) | 14.75 | 14.75 |
| South (J2) | 14.75 | 10.30 |
| East (J3) | 14.75 | 12.50 |
| West (J4) | 14.75 | 12.50 |

We then physically measured and verified that these new offsets reflect reality.

Electronics group will take out the jaw set with the view to recomissioning/cleaning it during the next shutdown. They will also look into adding anti-clash limit switches.

**Comment by davidkeymer** [comment]
Asked for update from current local contact (not original) and it was reported that the jaws are behaving well since recalibration.  This ticket can therefore be considered reviewed.

**NB** Apparently this _shift_ has happened before, after the jaw controllers have been powered back on, but not since SECI times.

**Comment by DominicOram** [comment]
From Jamie: Would have been easier to fix this if https://github.com/ISISComputingGroup/IBEX/issues/4390 was done.
